src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```